src/inputs/dropdown/dropdown.jsx

```diff
@@ -71,6 +71,37 @@ class Dropdown extends React.Component {
         if (!_.isEqual(currentPropsValue, previousPropsValue)) {
             this.setState({ menuIsOpen: false, value: currentPropsValue });
         }
+
+        const {
+            autoScrollSelection,
+            options,
+            selection,
+            selectionMatchProp,
+            value
+        } = this.props;
+        if (selection && autoScrollSelection && this.dropdownMenu && value) {
```

Comment:
Nitpick: I personally prefer an empty line in between two lines like these, but this is absolutely _Not a Blocker_

Reply:
Me too, but I agree NAB.

Reply:
Looks good.  Resolving.